core/templates/dev/head/components/ExplorationCreationService.js

```diff
@@ -0,0 +1,128 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Functionality for the create exploration button and upload
+ * modal.
+ */
+
+// Service for the create/upload exploration buttons and modals.
```

Comment:
Nit: Probably worth deleting this. The fileoverview already specifies this information.